### PR TITLE
Modal component added

### PR DESCRIPTION
**Pull request by anand-tiwari**
To see modal component please open netlify app link mentioned, it will open that component page directly.

### CODE REVIEW

**Comment by anshul-bheniye** [comment]
It will help reviewers if you add screenshot or gif of what you did.

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="modal__title">Modal Title (in center )</div>
+            <div class="modal__close"><i class="fa fa-times" aria-hidden="true"></i></div>
+        </div>
+        <div class="modal__content">
+            <div class="modal__body">
+                Are you sure you want to delete this message?
+            </div>
+            <div class="modal__footer">
+                <button class="cs-btn cs-btn-primary">cancle</button>
+                <button class="cs-btn cs-btn-primary">delete</button>
```
Comment (anshul-bheniye) [inline components/modal/modal.html:13]: line 12, line 30 and line 47 have typo.

**Review by anshul-bheniye — APPROVED** [review]
overall code looks good

**Comment by anand-tiwari** [comment]
> 

if you open netlify app link it will open that component page directly.

**Pull request merged**